doc/samtools-ampliconclip.1

```diff
@@ -113,7 +115,8 @@ Soft clip reads (default).
 Hard clip reads.
 .TP
 .B --both-ends
-Clip at both the 5' and the 3' ends where regions match.
+Clip at both the 5' and the 3' ends where regions match.  Negates the --strand
```

Comment:
Is "negates" the right term to use here?  Maybe say "\fB--strand\fR is ignored when using this option"?

Reply:
Changed the wording.